dotnet/src/Connectors/Connectors.AzureAISearch.UnitTests/AzureAISearchCollectionCreateMappingTests.cs

```diff
@@ -214,7 +214,6 @@ public static IEnumerable<object[]> DataTypeMappingOptions()
         yield return new object[] { typeof(long), SearchFieldDataType.Int64 };
         yield return new object[] { typeof(float), SearchFieldDataType.Double };
         yield return new object[] { typeof(double), SearchFieldDataType.Double };
-        yield return new object[] { typeof(DateTime), SearchFieldDataType.DateTimeOffset };
```

Comment:
/cc @westey-m I think this is part of the more general timestamp discussion we need to have...

Reply:
Ah, @adamsitnik pointed out to me that we already don't support DateTime (the model builder throws), so this is just a cleanup. Looks OK in that case.